### PR TITLE
WIP reproducer for CE startup error that doesn't prevent app startup

### PR DESCRIPTION
**Pull request by henryju**
DO NOT MERGE !!!

### CODE REVIEW

File: server/sonar-ce/src/main/java/org/sonar/ce/app/CeServer.java [hunk 1]
```diff
@@ -139,6 +139,11 @@ public CeMainThread() {
     public void run() {
       boolean startupSuccessful = attemptStartup();
       this.started = true;
+      try {
+        Thread.sleep(2000);
+      } catch (InterruptedException e) {
```
Comment (SonarTech) [inline server/sonar-ce/src/main/java/org/sonar/ce/app/CeServer.java]: MAJOR Either re-interrupt this method or rethrow the "InterruptedException". [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2142)

File: server/sonar-ce/src/main/java/org/sonar/ce/app/CeServer.java [hunk 2]
```diff
@@ -139,6 +139,11 @@ public CeMainThread() {
...
+        e.printStackTrace();
```
Comment (SonarTech) [inline server/sonar-ce/src/main/java/org/sonar/ce/app/CeServer.java]: MINOR Use a logger to log this exception. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1148)

**Comment by SonarTech** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: server/sonar-ce/src/main/java/org/sonar/ce/app/CeServer.java [hunk 3]
```diff
@@ -138,7 +139,13 @@ public CeMainThread() {
     @Override
     public void run() {
       boolean startupSuccessful = attemptStartup();
+      this.operational = startupSuccessful;
       this.started = true;
+      try {
+        Thread.sleep(2000);
+      } catch (InterruptedException e) {
```
Comment (SonarTech) [inline server/sonar-ce/src/main/java/org/sonar/ce/app/CeServer.java]: MAJOR Either re-interrupt this method or rethrow the "InterruptedException". [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2142)

**Comment by sns-seb** [comment]
@henryju thanks for the reproducer. This is fixed in #2644

**Pull request closed without merging**